demo/admin/src/products/ProductForm.gql.ts

```diff
@@ -1,5 +1,15 @@
 import { gql } from "@apollo/client";
 
+// TODO: import from "@comet/admin" once the types are generated correctly in `ProductForm.gql.generated.ts`
+// import { finalFormFileUploadFragment } from "@comet/cms-admin";
```

Comment:
Why doesn't this work? We can have a look at it together

Reply:
I just had a look at it: The type generation doesn't work because the GraphQL code generator doesn't know about the fragment defined in the library. Adding it to the config (`documents: [“./src/**/!(*.generated).{tsx,ts}“, “../../packages/admin/cms-admin/src/**/!(*.generated).{tsx,ts}“],`) works, but would need to be done manually in every project. I'm not sure about that. 

Possible alternatives:

1. Re-export the generated type in the package (GQLFinalFormFileUploadFragment) and use it in the application
2. Don't use fragments in the library at all

@nsams @thomasdax98 what's your opinion?

Reply:
:one: seems a good idea

Reply:
Like this? c225f75bb75f14d3b488c3b237476e0641210522

Reply:
It's better, but having to define the type manually where the fragment is used isn't ideal.

Reply:
True but I'm not sure how else to do it 😅 

IMO it makes most sense, to add all fragments, that are imported from another package to the config (the `documents` array). If there is a way to do this automatically it would be perfect 🤔

Reply:
You could try if adding `@comet/cms-admin/dist/**/*.js` to `documents` works

Reply:
No, doesn't change anything. 

I added it here, under `"./src/" -> "documents"`: [demo/admin/codegen.ts#L35](https://github.com/vivid-planet/comet/blob/next/demo/admin/codegen.ts#L35)
Maybe this doesn't work, as `@comet/cms-admin` is not inside `./src` 🤔 

Do we maybe need an additional entry in the `generates` array?

Reply:
> Do we maybe need an additional entry in the generates array?

No, this wouldn't fix the issue

@nsams do you have an idea?

Reply:
And something the Admin Generator should generate?